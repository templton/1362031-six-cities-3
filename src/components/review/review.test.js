import React from "react";
import renderer from "react-test-renderer";
import Review from "./review";
import Proptypes from "prop-types";

const reviewMock = {
  userName: "Ivan",
  avatar: "",
  timestamp: "2020-03-20 00:00:00",
  rating: 4,
  comment: "Commnet text"
};

describe(`Render <Review/>`, () => {
  it(`Default render`, () => {
    const tree = renderer
      .create(<Review reviewDetail={reviewMock}/>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
