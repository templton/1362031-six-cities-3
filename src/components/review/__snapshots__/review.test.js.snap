// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <Review/> Default render 1`] = `
Array [
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src=""
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Ivan
    </span>
  </div>,
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
      <span
        className="reviews__rating-value rating__value"
      >
        4
      </span>
    </div>
    <p
      className="reviews__text"
    >
      Commnet text
    </p>
    <time
      className="reviews__time"
      dateTime="2020-03-20 00:00:00"
    >
      март 2020 г.
    </time>
  </div>,
]
`;
